src/models/participant/participantProxy.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const Db = require('../../lib/db')
+const ErrorHandler = require('@mojaloop/central-services-error-handling')
+
+exports.create = async (participantId, isProxy) => {
+  try {
+    const result = await Db.from('participantProxy').insert({
```

Comment:
I think I had asked that question during refinement of how to determine if a participant was a proxy and that it was left to the assignee to decide.

Adding the new field would be straightfoward, I was planning to do that.
But, then I thought what if in the future there is something that is more proxy specific that doesn't apply to all participants that we wouldn't have a clean place to put that field.

I really did have 2nd thoughts that I might be overthinking this implementation.

Lmk what you prefer.

Reply:
I prefer new field in the same table because we don't need to change the logic in other places like getting endpoints of the participants, liquidity checks, position adjustments. All the other tables are foreign keyed with participants table.

Reply:
ok. I'll make the change